dbt/models/default/schema/default.vw_pin_sale.yml

```diff
@@ -14,7 +14,7 @@ models:
       - name: is_multisale
         description: '{{ doc("shared_column_sale_is_multisale") }}'
       - name: is_mydec_date
-        description: Indicator for whether or not the observation uses the myDec sale date
+        description: Indicator for whether or not the observation uses the MyDec sale date
```

Comment:
Just capitalizing MyDec correctly in all places in the documentation. Didn't want to put it in a separate PR.